### PR TITLE
Fix 3.0 branch building (WIP)

### PR DESCRIPTION
**Pull request by lzap**
Do not merge, need to test guides building for 3.0

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -42,6 +42,9 @@ jobs:
           ruby-version: ${{ env.RUBY_VERSION }}
           bundler-cache: true
 
+      - name: Run bundle install
+        run: bundle install
+
```
Comment (lzap) [inline .github/workflows/deploy.yml:47]: I don't understand why but in the 3.0 and 2.5 branches build fails with "bundle exec - not found". Ruby somehow installs correctly. I tried to copy the very same `deploy.yml` over from master but it just fails.

**Comment by lzap** [comment]
I am giving up, putting "bundle install" there for now, it is installing gems so apparently the ruby/setup-ruby does not somehow work or installs it incorrectly. Not sure maybe @ekohl can spot something later on.

**Comment by lzap** [comment]
Must be something with Gemfile or Gemfile.lock but I doublechecked that and they were the same.

**Comment by ekohl** [comment]
It can be that psych is in the gem cache. You can try https://github.com/ruby/setup-ruby#dealing-with-a-corrupted-cache

**Pull request merged**